raster/src/main/scala/geotrellis/raster/geotiff/GeoTiffResampleRasterSource.scala

```diff
@@ -130,6 +130,8 @@ class GeoTiffResampleRasterSource(
       ).resample(targetRasterExtent.cols, targetRasterExtent.rows, method)
     }
   }
+
+  override def toString: String = s"GeoTiffResampleRasterSource(${dataPath.value},$resampleTarget,$method)"
```

Comment:
GeoTiffResampleRasterSource will have an additional parameter OverviewStrategy in #3200. Should we add that to this method as well?